resume.md

```diff
@@ -1,49 +1,58 @@
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Android Developer**: Crowd Dynamics Lab   | Remote *September 2020 - Present*
+* Assist in the implementation of an Android tracer app using IoT principles
+
+
+**Backend Developer**: Ensemble Education | Remote *August 2020 - October 2020*
+* Implemented social media logins, file systems and assisted in building REST APIS in NodeJS and C++ 
+* Assisted in the development of the AI module using Python using scikit-learn, Pandas and Numpy.
+
+**Software Research Assistant**: UIUC advertising department   | Remote *July 2020 - August 2020*
+* Developed an AI chatbot in Node.js with a user interface in HTML/CSS. Has a user base of 200 surveyors.
+* Implemented a recognition and recommendation system for analysis and stored interactions in a database.
+
+**Undergraduate Data Science Researcher**: Yun Research Lab | Remote... *May 2020 - Augusr 2020*
+* Implemented features in the Social Media Macroscope for data analytics using Pandas, Numpy and scipy.
+* Built object detection programs in Python for image processing utilizing YOLOV3 and cleaned datasets
+
+
 
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
 
 ## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
+**Reddit bot**: *Makes requests to rate my professor and UIUC course APi and posts comments on reddits with relevant course info* |https://tinyurl.com/yyd39fs3 *August 2020*
+* Developed Reddit Professor in order to help students plan out classes
+* Beta Tested to thirthy users and collected Crash Data using Google analytics 
+
+**AI minesweeper game**: *Play minesweeper against the computer or make AI play for you* | https://tinyurl.com/y3h2zzug *June 2020*
+* Built using Python and path searching algorithms to find the nearest mines 
```

Comment:
Be specific, what components from pygame did you use?  What path seraching algorithms did you use?  BFS, DFS?